docs/developer-docs/multi-chain/ethereum/evm-rpc/samples.mdx

```diff
@@ -0,0 +1,19 @@
+---
+keywords: [advanced, ethereum, evm rpc, evm, tutorial]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# EVM RPC samples
+
+<MarkdownChipRow labels={["Advanced", "Ethereum", "Tutorial"]} />
+
+Here are some example projects using the EVM RPC canister:
+
+- [Vite + React + Motoko  + EVM RPC](https://github.com/rvanasa/vite-react-motoko/tree/evm-rpc)
+
+- [Vite + React + Rust + EVM RPC](https://github.com/fxgst/evm-rpc-rust/tree/main)
+
+- [ICP ETH Coprocessor](https://github.com/domwoe/icp-eth-coprocessor/tree/main)
+
+- [ckICP airdrop canister](https://github.com/benjizhai/ckicp-ethereum-contracts)
```

Comment:
those are jut solidity contracts, there is no usage of the evm rpc canister so we should remove it. instead we can add those
- https://github.com/letmejustputthishere/chainfusion-starter/tree/main
- https://gitlab.com/b4b-world/b4b-icp

Reply:
Added this examples and removed the Airdrop canister example